arches_component_lab/src/arches_component_lab/widgets/RichTextWidget/components/RichTextWidgetEditor.vue

```diff
@@ -0,0 +1,104 @@
+<script setup lang="ts">
```

Comment:
These files are exact dupes for now. Should we just remove them (is it okay to delay supporting this widget)?

Reply:
I think it's ok to delay this widget.  Is it hard to incorporate this primevue component https://primevue.org/editor/